packages/frontend/src/pages/Users/components/UsersTable.tsx

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
-import { useRecoilValue } from 'recoil';
+import { useRecoilValue, useRecoilState } from 'recoil';
 import { UserDto, UserRole } from 'api/dist/user/types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleXmark } from '@fortawesome/free-solid-svg-icons';
 import {
   AllAdminUsers,
   AllForwarderUsers,
   AllQuantifierUsers,
   AllUsers,
+  UsersTableData,
+  UsersTableSelectedRole,
+  UsersTableFilter,
+  UsersTablePage,
+  UsersTableLastPage,
 } from '@/model/users';
 import { SearchInput } from '@/components/form/SearchInput';
 import { SelectInput } from '@/components/form/SelectInput';
 import { UsersTableRow } from './UsersTableRow';
 import { UsersTablePagination } from './UsersTablePagination';
 
-interface roleOptionsProps {
-  value: string;
-  label: string;
-}
-
 const roleOptions = [
```

Comment:
You are maintaining two roleOptions lists. One here and one in `model/users.ts`. One should be enough. Consider removing this list and exporting the one found in `model`.